src/DataProtection/DataProtection/src/AuthenticatedEncryption/ManagedAuthenticatedEncryptorFactory.cs

```diff
@@ -127,5 +127,14 @@ private class AlgorithmActivatorCore<T> : IActivator<T> where T : new()
                 public Func<T> Creator { get; } = Activator.CreateInstance<T>;
             }
         }
+
+        private partial class Log
+        {
+            [LoggerMessage(10, LogLevel.Debug, "Using managed keyed hash algorithm '{FullName}'.", EventName = "UsingManagedKeyedHashAlgorithm")]
+            public static partial void UsingManagedKeyedHashAlgorithm(ILogger logger, string fullName);
+
+            [LoggerMessage(11, LogLevel.Debug, "Using managed symmetric algorithm '{FullName}'.", EventName = "UsingManagedSymmetricAlgorithm")]
```

Comment:
This is a good example where everything is using a shared logger and it's hard to ensure the ids are unique. What was the conclusion there? Moving the IDs back to a shared class would be helpful.

Reply:
> and it's hard to ensure the ids are unique

The source generator ensures are ids are unique for a given log scope so we think that's good enough. 

In particular, the only time moving the shared one to a scoped one is problematic is when a previously used id is no longer used and consequently can become accidentally re-used in the future. CORS had a couple of these: https://github.com/dotnet/aspnetcore/pull/34699/files#diff-a39be06054f8e5592a940c4591a961448a0f9f0c6c36cd564ba8c0239414f99dR289-R290

Reply:
How can it tell these are in the same log scope as adjacent classes? It only references an ILogger here.

Reply:
I was wrong about this instance being shared, but I've found a few more shared that I'll flag.

Reply:
Can we just leave all the logs that were in one file together? It just seems simpler and less confusing than to force a new pattern afterwards.